Document specific Java version required for Clouseau

## Description
FYI, this is not the same as [Issue 2630](https://github.com/apache/couchdb/issues/2630). I got passed that stage already and was able to start Clouseau. Anyway, I followed the documentation of CouchDB 3.0 to set up a single node server on Windows 2k8. Clouseau was also set up according to section **1.8. Search Plugin Installation** in the documentation. But I still couldn't get Clouseau connected to CouchDB. When I **GET /** , the list of features returned did include "search", but when I actually performed a search, an error would be returned saying:

> Search Failed: Could not connect to the Clouseau Java service at clouseau@127.0.0.1

## Steps to Reproduce

1. Download CouchDB 3 for Windows
2. Follow the documentation to install and set up a single node server
3. Follow section **1.8. Search Plugin Installation** to download Clouseau and run
4. Add a simple index to test.

## Expected Behaviour

Test search returns result

## Your Environment

* CouchDB version used: 3.0.0
* Operating system and version: Windows 2008 (64bit)
* Clouseau version: 2.17.0
* Java SDK version: 14 (64bit)

## Additional Context
CouchDB **local.ini**
```
[couchdb]
single_node=true

[dreyfus]
name = clouseau@127.0.0.1
limit = 10
retry_limit = 5
```

In addition, the following config values was changed through Fauxton:
```
[chttpd]
bind_address = 0.0.0.0
[httpd]
bind_address = 0.0.0.0
```


CouchDB vm.args
```
# The name is the only change that I've made to the original vm.args
# The original value was couchdb@localhost
-name couchdb@127.0.0.1

-setcookie monster
-kernel error_logger silent
-sasl sasl_error_logger false
+K true
+A 16
+Bd -noinput
-smp enable
-ssl session_lifetime 300
```

clouseau.init
```
[clouseau]
name=clouseau@127.0.0.1
cookie=monster
dir=index
max_indexes_open=500
```

clouseau output
```
C:\Documents\Java\clouseau>java -server -Xmx2G -Dsun.net.inetaddr.ttl=30 -Dsun.n
et.inetaddr.negative.ttl=30 -Dlog4j.configuration=file:log4j.properties -cp "lib
\*" com.cloudant.clouseau.Main clouseau.ini

2020-04-04 16:25:51,496 org.apache.commons.configuration.ConfigurationUtils [DEB
UG] ConfigurationUtils.locate(): base is null, name is clouseau.ini

2020-04-04 16:25:51,497 org.apache.commons.configuration.DefaultFileSystem [DEBU
G] Could not locate file clouseau.ini at null: no protocol: clouseau.ini

2020-04-04 16:25:51,503 org.apache.commons.configuration.ConfigurationUtils [DEB
UG] Loading configuration from the path clouseau.ini

2020-04-04 16:25:51,745 org.jboss.netty.channel.socket.nio.NioProviderMetadata [
DEBUG] Using the autodetected NIO constraint level: 0

2020-04-04 16:25:51,913 clouseau.main [INFO] Clouseau running as clouseau@127.0.
0.1

2020-04-04 16:25:51,931 scalang.ErlangNode [DEBUG] deliverMonitor Pid('clouseau@
127.0.0.1,2,1,3) -> Pid('clouseau@127.0.0.1,3,1,3) (Reference('clouseau@127.0.0.
1,WrappedArray(0, 0, 0),3))

2020-04-04 16:25:51,936 scalang.ErlangNode [DEBUG] deliverMonitor Pid('clouseau@
127.0.0.1,2,1,3) -> Pid('clouseau@127.0.0.1,4,1,3) (Reference('clouseau@127.0.0.
1,WrappedArray(1, 0, 0),3))

2020-04-04 16:25:51,938 scalang.ErlangNode [DEBUG] deliverMonitor Pid('clouseau@
127.0.0.1,2,1,3) -> Pid('clouseau@127.0.0.1,5,1,3) (Reference('clouseau@127.0.0.
1,WrappedArray(2, 0, 0),3))

2020-04-04 16:25:58,743 scalang.node.ErlangHandler [DEBUG] handler message Hands
hakeSucceeded('couchdb@127.0.0.1,[id: 0x73d0b7f5, /127.0.0.1:64046 => /127.0.0.1
:64026])

2020-04-04 16:25:58,756 scalang.node.ErlangHandler [DEBUG] handler message Monit
orMessage(Pid('c o u c h d b @ 1,14543,1,1),'n e t _ k ,Reference('c o u c h d b
 @ 1,Vector(53442, -689963005, 548569663),1))


2020-02-04 16:25:58,758 scalang.ErlangNode [DEBUG] monitor Pid('c o u c h d b @
1,14543,1,1) -> 'n e t _ k  (Reference('c o u c h d b @ 1,Vector(53442, -6899630
05, 548569663),1))

2020-02-04 16:25:58,762 scalang.ErlangNode [DEBUG] pids List(Pid('clouseau@127.0
.0.1,3,1,3), Pid('clouseau@127.0.0.1,5,1,3), Pid('clouseau@127.0.0.1,1,1,3), Pid
('clouseau@127.0.0.1,2,1,3), Pid('clouseau@127.0.0.1,0,0,3), Pid('clouseau@127.0
.0.1,4,1,3))

2020-02-04 16:25:58,763 scalang.ErlangNode [WARN] Try to monitor non-live proces
s: Pid('c o u c h d b @ 1,14543,1,1) -> 'n e t _ k  (Reference('c o u c h d b @
1,Vector(53442, -689963005, 548569663),1))

2020-02-04 16:25:58,764 scalang.ErlangNode [DEBUG] handling monitor exit for Mon
itor(Pid('c o u c h d b @ 1,14543,1,1),'n e t _ k ,Reference('c o u c h d b @ 1,
Vector(53442, -689963005, 548569663),1))

2020-02-04 16:25:58,766 scalang.ErlangNode [DEBUG] node scalang.ErlangNode@570f5
ff4 sending MonitorExitMessage('n e t _ k ,Pid('c o u c h d b @ 1,14543,1,1),Ref
erence('c o u c h d b @ 1,Vector(53442, -689963005, 548569663),1),'noproc)

2020-02-04 16:25:58,776 scalang.epmd.EpmdHandler [DEBUG] Uh oh disconnect.

2020-02-04 16:25:58,776 scalang.epmd.EpmdHandler [DEBUG] Oh snap channel closed.


2020-02-04 16:26:03,783 scalang.epmd.EpmdHandler [DEBUG] Uh oh disconnect.

2020-02-04 16:26:03,784 scalang.epmd.EpmdHandler [DEBUG] Oh snap channel closed.


2020-02-04 16:26:08,787 scalang.node.ErlangHandler [DEBUG] handler message RegSe
nd(Pid('c o u c h d b @ 1,14543,1,1),'n e t _ k ,('$ g e n _,(Pid('c o u c h d b
 @ 1,14543,1,1),Reference('c o u c h d b @ 1,Vector(53442, -689963005, 548569663
),1)),('i s _ a,'c o u c h d b @ 1)))

2020-02-04 16:26:08,790 scalang.node.ErlangHandler [ERROR] error caught in erlan
g handler 'couchdb@127.0.0.1
java.lang.Exception: EPMD Registration timed out.
        at scalang.epmd.EpmdHandler$EpmdResponse.call(EpmdHandler.scala:85)
        at scalang.epmd.Epmd.lookupPort(Epmd.scala:89)
        at scalang.ErlangNode.connectAndSend(Node.scala:877)
        at scalang.ErlangNode$$anonfun$4.apply(Node.scala:859)
        at scalang.ErlangNode$$anonfun$4.apply(Node.scala:859)
        at overlock.atomicmap.OneShotThunk.value(OneShotThunk.scala:32)
        at overlock.atomicmap.OneShotThunk.equals(OneShotThunk.scala:35)
        at org.cliffc.high_scale_lib.NonBlockingHashMap.remove(NonBlockingHashMa
p.java:334)
        at overlock.atomicmap.AtomicMap.getOrElseUpdate(AtomicMap.scala:77)
        at scalang.ErlangNode.getOrConnectAndSend(Node.scala:858)
        at scalang.ErlangNode.monitorExit(Node.scala:631)
        at scalang.ErlangNode.monitorWithoutNotify(Node.scala:604)
        at scalang.node.ErlangHandler.messageReceived(ErlangHandler.scala:61)
        at scalang.node.FailureDetectionHandler.messageReceived(FailureDetection
Handler.scala:33)
        at scalang.node.PacketCounter.messageReceived(PacketCounter.scala:17)
        at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:30
2)
        at org.jboss.netty.handler.codec.oneone.OneToOneDecoder.handleUpstream(O
neToOneDecoder.java:76)
        at scalang.node.PacketCounter.messageReceived(PacketCounter.scala:17)
        at org.jboss.netty.channel.Channels.fireMessageReceived(Channels.java:30
2)
        at org.jboss.netty.handler.codec.frame.FrameDecoder.unfoldAndFireMessage
Received(FrameDecoder.java:321)
        at org.jboss.netty.handler.codec.frame.FrameDecoder.callDecode(FrameDeco
der.java:299)
        at org.jboss.netty.handler.codec.frame.FrameDecoder.messageReceived(Fram
eDecoder.java:216)
        at org.jboss.netty.handler.execution.ChannelEventRunnable.run(ChannelEve
ntRunnable.java:69)
        at org.jboss.netty.handler.execution.OrderedMemoryAwareThreadPoolExecuto
r$ChildExecutor.run(OrderedMemoryAwareThreadPoolExecutor.java:316)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoo
lExecutor.java:1130)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPo
olExecutor.java:630)
        at java.base/java.lang.Thread.run(Thread.java:832)
        at overlock.threadpool.ErrorLoggedThread.run(NamedThreadFactory.scala:40
)

2020-02-04 16:26:08,800 scalang.node.ErlangHandler [INFO] channel disconnected o
rg.jboss.netty.channel.DefaultChannelPipeline$DefaultChannelHandlerContext@41b86
681 [id: 0x73d0b7f5, /127.0.0.1:64046 :> /127.0.0.1:64026] DISCONNECTED. peer: '
couchdb@127.0.0.1

```

**NOTE:** As soon as the exception was thrown in **Clouseau**, **GET /** from **CouchDB** would not show "search" as a feature.

It seems that CouchDB can see the presence of Clouseau, but somehow Clouseau cannot talk back to CouchDB and time out.

As far as I can tell, I follow every step in the documentation and cannot see which part went wrong. How can I fix it?

--- a/src/install/search.rst
+++ b/src/install/search.rst
@@ -26,7 +26,8 @@ service is installed on the same host as CouchDB and communicates with it over
 the loopback network.
 
 The search plugin is runtime-compatible with Java JDKs 6, 7 and 8. Building a
-release from source requires JDK 6.
+release from source requires JDK 6. **It will not work with any newer version of
+Java.** Sorry about that.
 
 Installation of Binary Packages
 ===============================

